docs/guides/composition-revisions-example.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Composition Revision Example
+weight: 101
+---
+This tutorial discusses how CompositionRevisions work and how they manage Composite Resource
+(XR) updates. This start with a `Composition` and `CompositeResourceDefinition` that defines a `MyVPC`
+resource.
+
+This tutorial creates multiple XRs and modifies Composition to apply different Composition Revisions to the XRs.
+
+## Install Crossplane
+1. Install Crossplane v1.11.0 or later.
+```shell
+kubectl create namespace crossplane-system
+helm repo add crossplane-master https://charts.crossplane.io/master/
+helm repo update
+helm install crossplane --namespace crossplane-system crossplane-master/crossplane --devel --version 1.11.0-rc.0.108.g0521c32e
+kubectl get pods -n crossplane-system
+```
+Expected Output:
+```shell
+NAME                                       READY   STATUS    RESTARTS   AGE
+crossplane-7f75ddcc46-f4d2z                1/1     Running   0          9s
+crossplane-rbac-manager-78bd597746-sdv6w   1/1     Running   0          9s
+```
+
+## Apply a Composition
```

Comment:
What do you think about making this step 2 of a Preparation section? 
I believe it is easier when steps are grouped as much as possible and presented as numbered steps to follow.